scripts/pack-debug.sh

```
@@ -0,0 +1,44 @@
+systype=$(uname -s)
+build_type="debug"
+
+. $script_dir/fns.sh --source-only
+
+cd $project_dir;
+app_ver=$(node -p "require('./package.json').version")
+
+pack_dist() {
+  rm -rf ./tmp/*.zip && mkdir -p ./tmp/
+
+  local git_committish=$(git log --format="%h" -n 1)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)